con-base/src/main/java/ba/codecentric/base/service/ImageServiceImpl.java

```diff
@@ -0,0 +1,56 @@
+package ba.codecentric.base.service;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.UUID;
+
+@Service
+public class ImageServiceImpl implements ImageService {
+
+    @Value("${image.directory.path}")
+    private String directoryPath;
+
+    @Value("${image.timestamp.format}")
+    private String timestampFormat;
+
+    /**
+     * Saving an image from InputStream on the file system
+     * If the directory doesn't exist, it is created
+     * Name of the image is a timestamp and it is returned as a string
```

Comment:
I don't like comments, just because of this. If something is later changed, comments stays with missleading information. Naming and code by itself should be self-explanatory.